src/services/InterviewService.js

```diff
@@ -557,7 +557,7 @@ partiallyUpdateInterviewById.schema = Joi.object().keys({
         end: Joi.string().regex(InterviewConstants.Nylas.StartEndRegex).required(),
         start: Joi.string().regex(InterviewConstants.Nylas.StartEndRegex).required()
       })
-    ).required(),
+    ),
     startTimestamp: Joi.date().greater('now').when('status', {
       is: [InterviewConstants.Status.Scheduled, InterviewConstants.Status.Rescheduled],
       then: Joi.required(),
```

Comment:
At the moment `startTimestamp` and `endTimestamp` are required if we update status to `scheduled` or `rescheduled`.

Let's also make them optional the next way:
- if we update status to `scheduled` or `rescheduled`, then `startTimestamp` and `endTimestamp` should now allow `null` (but still optional). Otherwise, they may allow `null`.

Reply:
Done in latest commits.